pgmpy/independencies/Independencies.py

```diff
@@ -195,7 +195,14 @@ def __str__(self):
     __repr__ = __str__
 
     def __eq__(self, other):
-        return self.get_assertion() == other.get_assertion()
+        if not isinstance(other, IndependenceAssertion):
+            return False
```

Comment:
@yashu-seth The test cases in test suits aren't running this line of code and [this](https://github.com/pgmpy/pgmpy/pull/543/files#diff-0cea7b4ba2f4037dc81624c3791b3170R203) too.
Since less percentage of lines are being covered in tests so coverage decreases .